Makefile

```diff
@@ -216,9 +216,9 @@ test-e2e-sharded: build-all
 	TEST_IMAGE=$$(KO_DOCKER_REPO=kind.local ko build --platform=linux/$(ARCH) ./test/e2e/fixtures/kcp-test-image) && test -n "$${TEST_IMAGE}"; \
 	NO_GORUN=1 ./bin/sharded-test-server --v=2 --log-dir-path="$(LOG_DIR)" $(TEST_SERVER_ARGS) 2>&1 & PID=$$!; echo "PID $$PID"; \
 	trap 'kill -TERM $$PID' TERM INT EXIT; \
-	while [ ! -f .kcp/admin.kubeconfig ]; do sleep 1; done; \
+	while [ ! -f .kcp-0/admin.kubeconfig ]; do sleep 1; done; \
 	NO_GORUN=1 $(GO_TEST) -race -count $(COUNT) -p $(E2E_PARALLELISM) -parallel $(E2E_PARALLELISM) $(WHAT) \
-		-args --use-default-kcp-server --syncer-image="$${SYNCER_IMAGE}" --kcp-test-image="$${TEST_IMAGE}" --pcluster-kubeconfig="$(PWD)/kind.kubeconfig" $(TEST_ARGS)
+		-args --shard-kubeconfig=$(PWD)/.kcp-0/admin.kubeconfig --root-shard-admin-context="shard:admin" --syncer-image="$${SYNCER_IMAGE}" --kcp-test-image="$${TEST_IMAGE}" --pcluster-kubeconfig="$(PWD)/kind.kubeconfig" $(TEST_ARGS)
```

Comment:
better `--root-shard-kubeconfig`, no?

Reply:
ok, but why the `root` prefix? the kubeconfig contains both root, and base. actually the `root-shard-admin-context` should be renamed just to `shard-admin-context`, as in kubeconfig it `system:admin` context points to cluster `base` and user `shard-admin`.

Reply:
root in the sense of root shard, not root workspace.

kcp-0 is the root shard.

Reply:
ok, now this makes much more sense 👍

Reply:
fixed